packages/esm-patient-test-results-app/src/trendline-new/trendline.scss

```diff
@@ -0,0 +1,82 @@
+@import "~@carbon/charts/styles.css";
+@import "~@carbon/colors/scss/index.scss";
+@import "~@openmrs/esm-styleguide/src/vars";
+@import "carbon-components/scss/globals/scss/typography.scss";
+
+.ui-shell-header-modal-bar-default {
+  height: 3rem;
+  object-fit: contain;
+  background-color: #005d5d;
+  color: white;
+  padding-top: 1rem;
+  padding-left: 0.938rem;
+  display: flex;
+}
+
+.content {
+  display: flex;
+  justify-content: center;
+  align-items: baseline;
+  width: calc(100% - 22rem);
+}
+
+.title {
+  @include carbon--type-style('productive-heading-02');
+  color: $text-02;
+  margin-right: 4px;
+}
+
+.reference-range {
+  @include carbon--type-style('helper-text-01');
+  color: $text-05;
+}
+
+:global(.bx--cc--area path.area) {
+  stroke: none !important;
+  fill: #a7f0ba !important;
```

Comment:
Don't use important, and don't globally override a carbon class. This style sheet will affect all other apps trying to use the class whether they wanted to or not. Instead, make the global only change if it is a child of a trendline component. 

If you really intend to globally change the carbon classes then do it someplace higher in the css hierarchy, like in `openmrs-esm-core`

Reply:
Sure! I'll have a look at how can I refactor this.
Thanks!